sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/ConditionalRequest.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration;
+
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.data.appconfiguration.models.ConfigurationSetting;
+
+/**
+ * Sample demonstrates how to add, get, and delete a configuration setting by conditional request.
```

Comment:
…demonstrates how to conditionally add, get, and delete a configuration setting.